addon/components/ilios-calendar-event-month.js

```diff
@@ -11,13 +11,26 @@ export default CalendarEvent.extend({
   event: null,
   timeFormat: 'h:mma',
   classNameBindings: [':event', ':event-pos', ':ilios-calendar-event', 'event.eventClass', ':month-event'],
-  tooltipContent: computed('event', function(){
-    let str = this.get('event.location') + '<br />' +
-      moment(this.get('event.startDate')).format(this.get('timeFormat')) + ' - ' +
-      moment(this.get('event.endDate')).format(this.get('timeFormat')) + '<br />' +
-      this.get('event.name');
-  
-    return str;
+  tooltipContent: computed('event', function() {
+    const location = this.get('event.location');
+    const name = this.get('event.name');
+    const startTime = moment(this.get('event.startDate')).format(this.get('timeFormat'));
+    const endTime = moment(this.get('event.endDate')).format(this.get('timeFormat'));
+    const dueThisDay = this.get('dueThisDay');
+
+    let isILM = this.get('event.name').substring(0, 4) === 'ILM:';
+
+    if (isILM) {
+      if (location) {
+        return `${location}<br />${dueThisDay}<br />${name}`;
+      } else {
+        return `${dueThisDay}<br />${name}`;
+      }
+    } else if (isBlank(location)) {
+      return `Rm. TBD<br />${startTime} - ${endTime}<br />${name}`;
```

Comment:
Lets just go with TBD here instead of Rm. TBD.  We may have to fix the translations later, but for now just one english abbreviation at a time.